wg-naming/recommendations/template.md

```diff
@@ -0,0 +1,17 @@
+# Recommendation: Old Term -> New Term
+
+**Status:** Accepted|Draft|Other
+
+## Recommendation 
+
+- Formalize recommendation in title 
+- Provide a brief, 1-3 sentence summary of reasoning behind this change
+- Provide alternate recommendation(s) to the one in the title if needed 
+
+## Context/Precedents 
```

Comment:
Does this mean the context for the recommendation, or the context for the term in which this replacement makes sense?

Reply:
Context for the recommendation!